clients/python/sysadmin/helpers.py

```diff
@@ -0,0 +1,43 @@
+import copy
```

Comment:
Obviously this file is pretty innocent, but I don't think you should really include it here. I think providing an entry point for jinja2 helpers is important, but that should be customizable by the user